src/pages/Entries/ScrollableCalendar.tsx

```diff
@@ -142,7 +143,7 @@ export default function ScrollableCalendar({
                   <HappinessCard
                     key={selectedEntry?.id}
                     data={selectedEntry}
-                    click={() => { }}
+                    click={() => {}}
```

Comment:
bro we have different Prettier settings, we need to resolve this in dev meeting this week so I don't have to keep looking at formatting differences.